website/docs/project_settings/settings_project_global.md

```diff
@@ -45,6 +45,21 @@ The _input pattern_ matching uses REGEX expression syntax (try [regexr.com](http
 The **colorspace name** value is a raw string input and no validation is run after saving project settings. We recommend to open the specified `config.ocio` file and copy pasting the exact colorspace names.
 :::
 
+### Extract OIIO Transcode
+There is profile configurable (see lower) plugin which allows to transcode any incoming representation to one or multiple new representations (configured in `Output Definitions`) with different target colorspaces.
+Plugin expects instances with filled dictionary `colorspaceData` on a representation. This data contains information about source colorspace and must be collected for transcoding.
+`oiiotool` is used for transcoding, eg. `oiiotool` must be present in `vendor/bin/oiio` or environment variable `OPENPYPE_OIIO_PATHS` must be provided for custom oiio installation.
+
+Notable parameters:
+- **`Delete Original Representation`** - keep or remove original representation. If old representation is kept, but there is new transcoded representation with 'Create review' tag, original representation looses its 'review' tag if present.
+- **`Extension`** - target extension, could be empty - original extension is used
+- **`Colorspace`** - target colorspace - must be available in used color config
+- **`Display & View`** - transcoding into colorspace OR into display and viewer space could be used. (It is disjunctive: Colorspace & nothing in Display and View or opposite)
+- **`Arguments`** - special additional command line arguments for `oiiotool`
+
+
+Example here describes use case for creation of new color coded review of png image sequence. Original representation's files are kept intact, review is created from transcoded files, but these files are removed in cleanup process.
+![global_oiio_transcode](assets/global_oiio_transcode.png)
```

Comment:
Could we have another example here for transcoding just the review for a Maya beauty render?

Reply:
Think the main things would be:

- no `Colorspace` or `Display & View` cause that is taken from the DCC (Maya).
- subset regex to only transcode "beauty" render layer.

Reply:
How this should work? I don't have `Colorspace` collected from DCC